### DISCUSSION TITLE
how to change the default bundle directory

### DISCUSSION THREAD

**Discussion by beilunyang** [tauri-apps/tauri#5737] [Q&A] [answered]
Hi all,  i want to change the default bundle directory(src-tauri/target/release/bundle/), but i can't get a way to change. i have already read the official documentation

**Comment by FabianLars** [comment, marked as answer]
You can only change the `target/` dir via a cargo config (look for build.target: https://doc.rust-lang.org/cargo/reference/config.html), but the rest of the path `release/bundle/` will stay the same.

What's the use case here? Asking to evaluate if a config on Tauri's side makes sense or if we should just recommend using package.json scripts (something along the lines of `"tauri:build": "tauri build && mv src-tauri/target/release/bundle/ ~/bundle"`)
